packages/odyssey-storybook/.storybook/main.js

```diff
@@ -52,3 +41,33 @@ module.exports = {
     "@pxblue/storybook-rtl-addon",
   ],
 };
+
+function buildRules(rules) {
+  return rules
+    .reduce((memo, rule) => {
+      const testString = rule.test?.toString();
+      const isStyleLoader = /s?css/.test(testString);
+      const isScriptLoader = /(jsx?|tsx?)/.test(testString);
```

Comment:
I prefer being explicit in these cases. It's easier to tell exactly what we want:
```suggestion
      const isStyleLoader = /(css|scss)/.test(testString);
      const isScriptLoader = /(js|jsx|ts|tsx)/.test(testString);
```

It's more clear and less prone to someone messing it up in the future.